sysutils/cdrtools/Portfile

```diff
@@ -1,98 +1,205 @@
+                        ${worksrcpath}/TEMPLATES/Defaults.clang \
+                        ${worksrcpath}/TEMPLATES/Defaults.xcc \
+                        ${worksrcpath}/ved/ved.h
 }
 
-use_configure       no
-
-# hangs the same way as smake itself with both llvm-gcc-4.2 and Xcode 4.1's clang
-# https://trac.macports.org/ticket/30310
-compiler.blacklist  llvm-gcc-4.2 macports-llvm-gcc-4.2 {clang < 300}
-
-configure.cflags-append \
-                    -DNO_SCANSTACK \
-                    -Wno-error=implicit-function-declaration
-
-build.cmd           smake
-build.args          CC=${configure.cc} \
-                    COPTX="${configure.cflags} [get_canonical_archflags cc]" \
-                    DEFOSINCDIRS=${prefix}/include \
-                    LDOPTX="${configure.ldflags} [get_canonical_archflags ld]" \
-                    LDPATH=
-
-# smake does not support -j
-# Set build.jobs to -1 to disable MacPorts from adding the -j flag
-build.jobs          -1
-
-post-build {
-    # The build script doesn't notice when a program fails to build, so
-    # we must verify it ourselves. See
-    # https://trac.macports.org/ticket/34823
-    set progs {btcflash cdda2wav cdrecord devdump isodebug isodump isoinfo isovfy mkisofs readcd scgcheck scgskeleton}
-    fs-traverse path ${worksrcpath} {
-        if {[file isfile ${path}] && [string match {*/OBJ/*} ${path}]} {
-            set file [file tail ${path}]
-            set prog_index [lsearch ${progs} ${file}]
-            if {${prog_index} != -1} {
-                set progs [lreplace ${progs} ${prog_index} ${prog_index}]
-            }
-        }
-    }
-    if {[llength ${progs}] > 0} {
-        ui_error "The following programs did not build: [join ${progs} {, }]"
-        return -code error "build failed"
-    }
+conflicts_build-append  cdrtools
+
+compiler.blacklist      llvm-gcc-4.2 macports-llvm-gcc-4.2 {clang < 300}
+
+configure.ldflags-append -lintl
+
+#
+# Note:  The current version of smake in the MacPorts repo no longer builds the star port
+#        properly.  One must use a modern version of gnumake (port:gmake) instead.
+#
+
+#
+# Once a newer version of smake is merged into the MacPorts Repo,
+# Replace "port:gmake" with "port:smake" as a build dependency
+# and change build.cmd to "smake" and delete the pre-destroot block
+#
+
+# depends_build-append    port:smake \
+                          port:gettext-runtime \
+                          port:libiconv
+
+depends_build-append    port:gmake \
+                        port:gettext-runtime \
+                        port:libiconv
+
+depends_lib-append      port:gettext-runtime \
+                        port:libiconv
+
+# build.cmd               ${prefix}/bin/smake
+
+#
+# Use the ./Gmake script provided by the developer to build the port.
+# ./Gmake first builds a bootstrapped version of smake which is used to build
+# the rest of the port.
+#
+
+build.cmd               ${worksrcpath}/Gmake
```

Comment:
Just to verify, is this the correct case for the bundled `gmake`? (If not, this will fail on our buildbots, as we use a case-sensitive filesystem there.)

Reply:
> Just to verify, is this the correct case for the bundled `gmake`? (If not, this will fail on our buildbots, as we use a case-sensitive filesystem there.)

Checked the distfiles, and sure enough, `Gmake` is correct. All good!

Reply:
@macsguy -- I am updating this port given all your great feedback on the `smake` PR.  And yes the capital G in `./Gmake` is correct.  It is a script provided by the developer.  In any event, this statement will change!  since you just merged the `smake` PR.  

Because MacPorts now has an up to date version of `smake` in its repo,  this line will change to `build.cmd   smake`.   The `port.smake` is also being added as a build dependency and `port:gmake` is being removed.  I will do a rebase and force push VERY soon.